src/main/scala/zio/rocksdb/transaction/package.scala

```diff
@@ -0,0 +1,41 @@
+package zio.rocksdb
+
+import org.{ rocksdb => jrocks }
+import zio._
+
+package object transaction {
+  type Bytes         = Array[Byte]
+  type Transaction   = Has[RocksDB.TransactionService]
+  type TransactionDB = Has[RocksDB.TransactionDBService]
+
+  object RocksDB {
```

Comment:
This should be renamed to `TransactionDB` (as we’re dealing with the TransactionDB, after all, not RocksDB).

Reply:
It will conflict with https://github.com/zio/zio-rocksdb/pull/13/files#diff-c9ece5e61788085de4f97e406b1167a7R15

Reply:
We can call them `LiveTransactionDB` in that case.